<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_Shutter">
          <property name="text">
           <string>Shutter</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>mode filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Mode">
          <item>
           <property name="text">
            <string>eAlarmBlue                   </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eAlarmBlueHi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eGrayBW</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eGrayWB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eAlarmGreen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eIron</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eIronHi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eMedical</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eRainbow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eAlarmRed</string>
           </property>
          </item>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>mode scale </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Scale">
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>stop CAM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string> Temperature min-max </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_temperatureMIN">
            <property name="minimum">
             <number>-20</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>-20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_temperatureMAX">
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>manual min-max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_manualMIN">
            <property name="minimum">
             <number>-20</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_manualMAX">
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_posFocus">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="LabelVideo" name="labelVideo">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionimport_XML_settings"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionimport_XML_settings">
   <property name="text">
    <string>import *.XML settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelVideo</class>
   <extends>QLabel</extends>
   <header>labelvideo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
